assets/style.css

```diff
@@ -1,3 +1,4 @@
+@import "./nav.css";
```

Comment:
I think we can keep all the styles under a single file. This is a very simple webpage. No need for modularization at the moment.